ibacm/src/acm.c

```diff
@@ -2596,17 +2596,64 @@ acm_open_port(struct acmc_port *port, struct acmc_device *dev, uint8_t port_num)
 	port->state = IBV_PORT_DOWN;
 }
 
+static bool dev_support_ib(struct ibv_device *ibdev)
+{
+	struct ibv_context *verbs;
+	struct ibv_device_attr attr;
```

Comment:
... attr = {};

and you can remove the memset() below.

Reply:
I'd like to use memset to initialize a struct with zero. It is a clear statement for me.